Update AIShell docs for preview6 release (#337)
MicrosoftDocs/PowerShell-Docs-Modules: reference/ps-modules/AIShell/Invoke-AICommand.md

﻿---
external help file: AIShell.Integration.dll-Help.xml
Module Name: AIShell
ms.custom: 1.0.0-preview.6
ms.date: 08/25/2025
online version: https://learn.microsoft.com/en-us/powershell/module/aishell/invoke-aicommand?view=ps-modules&wt.mc_id=ps-gethelp
schema: 2.0.0
---

# Invoke-AICommand

## SYNOPSIS

Used by an MCP in AI Shell to invoke a command in the connected PowerShell session.

## SYNTAX

```
Invoke-AICommand [-Command] <ScriptBlock> [<CommonParameters>]
```

## DESCRIPTION

> This cmdlet is used by an MCP in AI Shell to invoke a command in the connected PowerShell session.
> This command is not intended to be used directly by users. When AI model creates a result containing
> commands, the MCP can use this cmdlet to execute the commands in the connected PowerShell session.

Reviewer: ```suggestion
MCPs in AI Shell use this cmdlet to invoke a command in the connected PowerShell session. This
command is not intended for users to directly invoke. When an AI model creates a result containing
commands, the MCP can use this cmdlet to execute the commands in the connected PowerShell session.
```
